lib/pages/labels/label.dart

```diff
@@ -7,12 +7,12 @@ class Label {
   static final dbColorCode = "colorCode";
   static final dbColorName = "colorName";
 
-  int id, colorValue;
-  String name, colorName;
+  int? id, colorValue;
+  String? name, colorName;
```

Comment:
`name`, `colorValue` and `colorName` are non-nullable. We can avoid `String?` here. This is will also help us to remove `!` force null conversion.